utils/notifications.py

```diff
@@ -61,7 +62,7 @@ def slack_notify(message, level=None):
             'debug' if (settings.ENV_DEV or settings.TEST) else 'info'
         )
         level = level if level in channels else default_level
-        channel = channels.get(level, htk_setting('HTK_SLACK_DEBUG_CHANNEL'))
+        channel = custom_channel or channels.get(level, htk_setting('HTK_SLACK_DEBUG_CHANNEL'))
```

Comment:
FWIW there is a Python HTK Lite that has the interface that I want: https://github.com/hacktoolkit/python-htk/blob/master/htk/utils/slack.py

Unfortunately that would take maybe a day's worth of refactoring or more (to ensure existing integrations don't break); longer term, being able to call `send_message` (from: https://github.com/hacktoolkit/python-htk/blob/fe1934124fff169269f93b58d08af06a9871246b/htk/utils/slack.py#L139) would be more elegant.